src/milind/SecondMaxFromArray.java

```diff
@@ -0,0 +1,24 @@
+package milind;
+
+public class SecondMaxFromArray {
+	
+	void secondMaxArrayNumber(int[] input) {
+		int temp;
+		for(int index = 0; index < input.length; index++) {
+			for(int innerIndex = index+1; innerIndex < input.length; innerIndex++) {

```

Comment:
you can handle in single loop only
check this out
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/924/files
take of this next time